include/zephyr/dt-bindings/gpio/adi-sdp-120.h

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2010-2024 Analog Devices Inc.
+ * Copyright (c) 2024 Baylibre, SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * @file
+ * @brief SDP-120 GPIO index definitions
+ *
+ * Defines meant to be used in conjunction with the "adi,sdp-120"
+ * ADI SDP-120 mapping.
+ *
+ * Example usage:
+ *
+ * @code{.dts}
+ * &spi1 {
+ *         cs-gpios = <&sdp_120 SDP_120_SPI_SS_N GPIO_ACTIVE_LOW>;
+ *
+ *         example_device: example-dev@0 {
+ *                 compatible = "vnd,spi-device";
+ *                 reg = <0>;
+ *         };
+ * };
+ * @endcode
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_ADI_SDP_120_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_ADI_SDP_120_H_
+
+/* GPIO */
+
+/**
+ * @brief IO[n] signal on a SDP-120 GPIO nexus node following
+ */
+
+#define SDP_120_IO(n) (n-1)
+
+	/* SPI */
```

Comment:
```suggestion
/* SPI */
```

This goes for all comments below as well.